ntoskrnl/mm/section.c

```diff
@@ -684,7 +684,7 @@ NTSTATUS NTAPI PeFmtCreateSection(IN CONST VOID * FileHeader,
     pssSegments[0].Length.QuadPart = nPrevVirtualEndOfSegment;
     pssSegments[0].RawLength.QuadPart = nFileSizeOfHeaders;
     pssSegments[0].Image.VirtualAddress = 0;
-    pssSegments[0].Image.Characteristics = IMAGE_SCN_CNT_INITIALIZED_DATA;
+    pssSegments[0].Image.Characteristics = 0;
```

Comment:
Does this make a difference in behavior?

Reply:
I did not check that, I simply went and removed all {{SCN_CNT}}

Reply:
Another option would be to just 'sanitize' Image.Characteristics, so that if `INITIALIZED_DATA` is set, we remove `UNINITIALIZED_DATA`

Reply:
I like the idea of removing it. I think the kernel is basically supposed to just ignore these flags completely.
In this instance, it's setting up a segment structure for the file header though, so initializing it with this flag seems reasonable to me (even though nothing should ever check the flag)